influxql/parser.go

```diff
@@ -1617,6 +1671,26 @@ func (p *Parser) parseSortField() (*SortField, error) {
 	return field, nil
 }
 
+// parseVarRef parses a reference to a measurement or field.
+func (p *Parser) parseVarRef() (*VarRef, error) {
+	// Save the current position in case we need to report an error.
+	_, pos, _ := p.s.curr()
+
+	// Parse the segments of the variable ref.
+	segments, err := p.parseSegmentedIdents()
+	if err != nil {
+		return nil, err
+	}
+
+	vr := &VarRef{Val: strings.Join(segments, ".")}
+
+	if len(segments) > 2 {
```

Comment:
Just to be clear, if we have more than 2 segments here, it's a problem? Normally we can have up to 3 segments. Which segment is not expected to be present?